### ISSUE TITLE
data_path issue

### ISSUE THREAD

**Issue by tengolagan** [Sohl-Dickstein/Diffusion-Probabilistic-Models#1]
Hi, 
trying to train the model on mnist, but I have the following error after running python train.py : 

> fuel.exceptions.ConfigurationError: Configuration not set and no default provided: data_path.

can't figure out how fuel should be set up.

thanks in advance for your help

**Comment by odellus** [comment]
Last release of fuel was in 2016. Might have better luck using a different framework than blocks + fuel tbh.

**Comment by Sohl-Dickstein** [comment]
Yeah, I also would strongly recommend building on something more modern. I would have had trouble debugging that error with an upstream library at the time I wrote the code ... and now ~ seven years later, I have no idea.

One good starting place I know of is Yang Song's github: https://github.com/yang-song/score_sde
